README.md

```diff
@@ -116,6 +116,28 @@ var expect = chai.expect;
 var should = chai.should();
 ```
 
+## Usage for ES6+
```

Comment:
OK, I [updated](https://github.com/chaijs/chai/pull/868/commits/4a19d5a646d4abeebdc2521851098fac143c632f) it like you suggested: